playwright/fixtures.mjs

```diff
@@ -0,0 +1,49 @@
+import { test as baseTest } from '@playwright/test';
+import fs from 'fs';
+import path from 'path';
+
+export * from '@playwright/test';
+
+export const test = baseTest.extend({}, { workerStorageState: '' });({
+  // Use the same storage state for all tests in this worker.
+  storageState: ({ workerStorageState }, use) => use(workerStorageState),
+
+  // Authenticate once per worker with a worker-scoped fixture.
+  workerStorageState: [
+    async ({ browser }, use) => {
+      // Use parallelIndex as a unique identifier for each worker.
+      const id = test.info().parallelIndex;
+      const fileName = path.resolve(test.info().project.outputDir, `.auth/${id}.json`);
+
+      if (fs.existsSync(fileName)) {
+        // Reuse existing authentication state if any.
+        await use(fileName);
+        return;
+      }
+
+      // Important: make sure we authenticate in a clean environment by unsetting storage state.
+      const page = await browser.newPage({ storageState: undefined });
+
+      // Perform authentication steps with your unique details.
+      await page.goto('http://localhost:5380/');
```

Comment:
This shouldn't be hardcoded.

Reply:
this page wrote initially now   i have never used it i have removed the whole page